spec/models/assignment_spec.rb

```diff
@@ -218,17 +218,17 @@
     end
 
     context 'when it is before the switchover' do
-      around do |example|
```

Comment:
rubocop-rspec-rails complains about `travel_to` in an `around` hook. This is because the `TimeHelper` methods always return to the present at the end of the test (via an `after_teardown` hook).